code-challenges.js

```diff
@@ -31,24 +36,74 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+
+// const boilingPoint = 212
+
+// const tempCheck = (num) => {
+//     if (num < boilingPoint){
+//         return `${num} is below boiling point`
+//     } else if ( num === boilingPoint){
+//         return `${num} is at boiling point`
+//     } else if (num > boilingPoint){
+//         return `${num} is above boiling point`
+//     }
+// }
+
+// console.log(tempCheck(temperature1))
+// console.log(tempCheck(temperature2))
+// console.log(tempCheck(temperature3))
+
+
+// output: 
+// 42 is below boiling point
+// 350 is above boiling point
+// 212 is at boiling point
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+//function
+// concat two arrays
+    //input: two arrays
+    //output/return: concatenatedArray.length   
+// 
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Pseudo code:
+const twoBaseballArrays = () => {
+    return padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns).length
+}
```

Comment:
This function has no parameters, which means it's not expecting any arguments. However, on line 80 in your console.log, you're giving the function 2 arguments. As it is right now, you can just console.log what you have on line 77 without creating a function and it will give you the same answer